packages/nest/src/schematics/library/library.ts

```diff
@@ -0,0 +1,140 @@
+import { normalize, Path } from '@angular-devkit/core';
+import {
+  apply,
+  chain,
+  externalSchematic,
+  filter,
+  MergeStrategy,
+  mergeWith,
+  move,
+  noop,
+  Rule,
+  SchematicContext,
+  template,
+  Tree,
+  url
+} from '@angular-devkit/schematics';
+import {
+  formatFiles,
+  getProjectConfig,
+  names,
+  offsetFromRoot,
+  toFileName,
+  updateJsonInTree,
+  updateWorkspaceInTree,
+  getNpmScope
+} from '@nrwl/workspace';
+import { Schema } from './schema';
+
+export interface NormalizedSchema extends Schema {
+  name: string;
+  prefix: string;
+  fileName: string;
+  projectRoot: Path;
+  projectDirectory: string;
+  parsedTags: string[];
+}
+
+export default function(schema: NormalizedSchema): Rule {
+  return (host: Tree, context: SchematicContext) => {
+    const options = normalizeOptions(host, schema);
+
+    return chain([
+      externalSchematic('@nrwl/workspace', 'lib', schema),
```

Comment:
Can you try this instead?
```suggestion
      externalSchematic('@nrwl/node', 'lib', schema),
```

The `@nrwl/node:lib` schematic includes a lot of the stuff that you have here, like handling tags, directories, test files, and adding the node builder to the architect.

Reply:
Great, that will clean it up greatly. I feel like there was a reason I went with workspace but it seems there is a lot I do not need if I go with `@nrwl/node` so I will spend some time switching it over tonight. Thanks